onboarding.md

```diff
@@ -31,293 +30,27 @@ That is it! Go to your onboarding issue and start checking things off!
-
-```
-Hey!
-
-Welcome to Niteo! We keep everything related to work on GitHub so I will be short:
-* read https://github.com/niteoweb/handbook/blob/master/onboarding.md,
-* open <LINK TO ONBOARDING ISSUE> and start at the top.
-
-
-Regards
-```
-
-
-
-# [Email Template] Niteo Email Account
-
-```
-Hey!
-
-Here's your Niteo email. You'll be using it for communication with us and for subscribing to websites.
-
-w: https://apps.rackspace.com/
-u: [email]
-p: [pass]
-
-Please change your password immediately after you login. You can do that in the top right Settings menu.
-
-Configure the email on your desktop app, find the instructions here: https://emailhelp.rackspace.com
-
-The next thing that you need to do is set up S/MIME email encryption. Generate a Comodo Certificate at https://www.comodo.com/home/email-security/free-email-certificate.php. You’ll need to Google on how to set encryption on your desktop app.
-
-This email is signed with my cert, you should be able to reply with an encrypted message, signed with your cert so that I can then reply back with an encrypted message as well.
-
-Once you have encryption ready, I’m sending over more information.
-
-
-Regards
-```
-
-
-# [Email Template] OpenVPN Access
-
-```
-Hey!
-
-Now that encryption is configured, you need to configure OpenVPN so you can connect to our internal services.
-
-Configuration files are attached, normally you just double-click them and follow instructions.
-
-When you manage to connect to our VPN you will then be able to access our internal services, such as http://docs.niteoweb.com/.
-
-
-Regards
-```
-
-# Email Signatures
-
-A few examples to base your signature off of. Choose your own title, it can be whatever you wish.
-
-```
-Richard Hendricks
-
-CEO
----------------
-Niteo.co
-
-E: rh@niteo.co
-W: www.niteo.co
-```
-
-```
-Erlich Bachmann
-
-Backend Specialist
-------------------
-Niteo.co
-
-E: eb@niteo.co
-W: www.niteo.co
-```
-```
-Monica Hall
-
-Senior (con)Sultan
-------------------
-Niteo.co
-
-E: mh@niteo.co
-W: www.niteo.co
-```
-
-# Offboarding
-
-```
-* [ ] Access to team 1Password.
-* [ ] Heroku access.
-* [ ] GitHub repositories access.
-* [ ] PyPICloud access.
-* [ ] Donations spreadsheet.
-* [ ] Slack access.
-* [ ] Email account.
-* [ ] Intra account.
-* [ ] VPN access.
-* [ ] Zoom.us account.
-* [ ] Sentry access.
-* [ ] EBN account.
-* [ ] DMON account.
-```
+* Add the new team member to the [@niteoweb/trialists] group so they get access to [operations](https://github.com/niteoweb/operations) and [handbook](https://github.com/niteoweb/handbook) repositories.
+* Create an [Trialist Onboarding Issue] and assign it to the new team member.
+* Send the welcome email, using the [Welcome  email template].
+* Create a `@niteo.co` email address and send credentials using the [Niteo account email template].
+* Send the OpenVPN details using the [OpenVPN access email template].
+
+Further useful template links:
+* [Intellectual Property Rights Disclaimer]
```

Comment:
"Further useful templates" ... hm ... we can do better.

- `[Intellectual Property Rights Disclaimer]` -> we need to tell the person going through onboarding that they need to sign it and instruct how to do it
- `[Full-time Onboarding Issue]` -> does this really need to be linked here? For how? Isn't it enough to just link it where you are instructing to create a new onboarding issue?
- same for offboarding

What I am trying to say: "useful" sections, "utills" fiels, etc. have a tendency to grow into abominations. So I like to avoid using them if possible. I.e. have a really good thought where they *really* fit instead of just putting them into "other useful things". If I cannot find a good fit, then maybe they are not needed?

Reply:
> we need to tell the person going through onboarding that they need to sign it and instruct how to do it

 IP contract is in checklist so yes, reference can be removed from this page.

> [Full-time Onboarding Issue] -> does this really need to be linked here?

There is no other referenc elsewhere as far as I am aware

> same for offboarding

again no reference elsewhere 

> have a tendency to grow into abominations ...  If I cannot find a good fit, then maybe they are not needed?

I agree with that however the full-time and offboarding templates are related to onboarding and creating new pages for them seemed unnecessary but still needed referenced. Perhaps they should be an additional two item to the peopleops list?

Reply:
No, not a new page. A paragraph maybe.

Reply:
I added them as extra steps to the end of the peopleops list, does this suffice?